spinalcordtoolbox/centerline/core.py

```diff
@@ -238,9 +241,22 @@ def get_centerline(im_seg, param=ParamCenterline(), verbose=1, remove_temp_files
                      else "centerline.nii.gz")
         im_centerline.save(os.path.join(tmp_folder, fname_ctr), mutable=True)
 
+    # If 'phys' is specified, adjust centerline coordinates (`Centerline.points`) and
+    # derivatives (`Centerline.derivatives`) to physical ("phys") space and native (`im_seg`) orientation.
+    if space == 'phys':
+        # Transform centerline to physical coordinate system
+        arr_ctl = im_seg.transfo_pix2phys([[x_centerline_fit[i], y_centerline_fit[i], z_ref[i]]
+                                           for i in range(len(x_centerline_fit))]).T
+        # Adjust derivatives with pixel size
+        nx, ny, nz, nt, px, py, pz, pt = im_seg.change_orientation(native_orientation).dim
```

Comment:
Small suggestion: We can indicate unused variables via `_`.

```suggestion
        _, _, _, _, px, py, pz, _ = im_seg.change_orientation(native_orientation).dim
```